downstream/snippets/snip-eda-25-plus-aap24.adoc

```diff
@@ -0,0 +1,3 @@
+You can configure a new installation of {EDAcontroller} 1.1 with {ControllerName} 4.4 or 4.5, which is considered a general avaliability release fully supported by Red Hat. {ControllerName} 4.4 and 4.5 were released as part of {PlatformNameShort} 2.4.
```

Comment:
Small typo: avaliability > availability

Reply:
Also, {ControllerName} > {ControllerNameStart} as its the start of a sentence

Reply:
This information will not be used as a snippet at this point, @g-murray